### ISSUE TITLE
Glue Bugged

### ISSUE THREAD

**Issue by Antillar** [Blood-Asp/GT5-Unofficial#880]
Glue is required for some recipes in GregTech. However, Glue seems to be bugged for me. 
To start with, the Glue itself has missing texture. This means that the texture is purple-black. Here is a screenshot showing this problem:
http://imgur.com/a/okNHv
Next, this bugged Glue is obtainable as normal in the Centrifuge, but it doesn't go in Assembling Machine, the Assembler simply doesn't recognise it as a valid fluid. 

I have GregTech v5.09.26 and Tinkers' Construct v1.8.8

**Comment by Blood-Asp** [comment]
Did you add tinkers later on in an already generated world?

Should that be the case, the "fix" would be the same as here: https://github.com/Blood-Asp/GT5-Unofficial/issues/577

**Comment by Antillar** [comment]
GregTech 5 was added later to the world as a GregTech 6 replacement when I have found out that GregTech 6 had no tech tree at that moment. 
In fluid registry only the glue from Tinkers' Construct was found and deleted. Launching the world afterwards changed all unusable GregTech glue to Tinkers' Construct glue, which successfully was accepted by assembling machine.

The problem was solved. The issue can be closed.
